Formula/pypy.rb

```diff
@@ -67,11 +78,15 @@ def install
       system python, buildpath/"rpython/bin/rpython",
              "-Ojit", "--shared", "--cc", ENV.cc, "--verbose",
              "--make-jobs", ENV.make_jobs, "targetpypystandalone.py"
+
+      with_env(PYTHONPATH: buildpath) do
+        system "./pypy-c", buildpath/"lib_pypy/pypy_tools/build_cffi_imports.py"
+      end
     end
 
     libexec.mkpath
     cd "pypy/tool/release" do
-      package_args = %w[--archive-name pypy --targetdir .]
+      package_args = %w[--no-make-portable --archive-name pypy --targetdir .]
```

Comment:
May need to check if `--no-make-portable` and/or `--no-embedded-dependencies` is correct option. Both are enabled on macOS:
```py
    parser.add_argument('--embedded-dependencies', '--no-embedded-dependencies',
                        dest='embed_dependencies',
                        action=NegateAction,
                        default=(ARCH in ('darwin', 'aarch64', 'x86_64')),
                        help='whether to embed dependencies in CFFI modules '
                        '(default on OS X)')
    parser.add_argument('--make-portable',
                        dest='make_portable',
                        action=NegateAction,
                        default=(ARCH in ('darwin',)),
                        help='make the package portable by shipping '
                            'dependent shared objects and mangling RPATH')
```

Our existing bottle has copied over tcl-tk/libsqlite dylibs, which means `tcl-tk` and `sqlite` are not runtime dependencies.
```console
❯ fd -e dylib .
lib/libpypy-c.dylib
libexec/lib/libpypy-c.dylib
libexec/lib/libsqlite3.0.dylib
libexec/lib/libtcl8.6.dylib
libexec/lib/libtk8.6.dylib
```

Reply:
Sadly, the arg is not exposed, so need to inreplace/patch it.